### ISSUE TITLE
Prompt dialog autofocus missing

### ISSUE THREAD

**Issue by PVince81** [owncloud/web#3684]
### Steps

1. Click "New" and pick "New folder"
1. Start typing
1. Click OK

### Expected result
Able to start typing right away, no backspace needed.

### Actual result
Cannot start typing. Need to first click on the input field.
Let's make such dialog automatically focus on the input field, which will also automatically select its contents (in the example case the contents says "New folder" which any user will likely want to replace)

### Version
Phoenix 0.11.0

@LukasHirt I think we talked about it and would have come with a recent ODS update related to modal dialogs.
Maybe it was a misunderstanding or it broke somehow.

**Comment by kulmann** [comment]
The focus is on the modal itself, not on the input, for accessibility reasons. Screen readers start where the focus is. So if you set the focus on the input element, the text context (label / description) before the input element is ignored by screen readers.

Current solution to mitigate this is that the input element is the first tabbable element (hit `TAB` once, then you're in the input element). But still, this is not a good solution.

We can use this ticket to track alternative solutions.

From @PVince81 
- solution 1: get rid of modal, do an inline rename
- solution 2: have accessibility preference to be able to switch this on

**Comment by labkode** [comment]
Hit was this, would be nice to have it fixed, is really appreciated from users to avoid that extra click :)

**Comment by kulmann** [comment]
Further discussion in https://github.com/owncloud/web/issues/3892#issuecomment-694918182 - needs to be solved in a WCAG 2.1 compliant way.
